### ISSUE TITLE
[BUG] method-default prefix is too restrictive

### ISSUE THREAD

**Issue by lmignon** [OCA/pylint-odoo#253]
This new check breaks the code if the provided value for a default is not a method or a lambda.

```python
default='section_1'`
```

```python
default=lambda m m._default_my_field()
```

**Comment by pedrobaeza** [comment]
Example of non-valid error?

**Comment by pedrobaeza** [comment]
Uhm, weird I haven't seen that problem on OCA Travis...

@moylop260 any clue?

**Comment by moylop260** [comment]
@blaggacao 
Could you check these valid cases of use for your new check, please?

**Comment by blaggacao** [comment]
> `default='section_1'`

this is intended to raise an error.

> `default=lambda m m._default_my_field()`

this is not... Hm..

**Comment by blaggacao** [comment]
Shouldn't it be: `default=lambda m: m._default_my_field()`, though?

**Comment by blaggacao** [comment]
The most canonical way would be:
`default=_default_my_field` (passed as symbol)

**Comment by lmignon** [comment]
@blaggacao 
`default=_default_my_field` is not overiddable since  you pass as symbol... IMO it's not the right way.

**Comment by blaggacao** [comment]
True it is... :wink:

**Comment by blaggacao** [comment]
Oh, I see. It was release, I'll take care of a fix asap... :fearful:

**Comment by lmignon** [comment]
yes it has been released :nauseated_face:

**Comment by moylop260** [comment]
So I will revert it and release the revert
In order to avoid making false reds for OCA projects

After fixes we can release a new version using all case of use from unittests
